content/shmem_alltoall_nb.tex

```diff
@@ -0,0 +1,123 @@
+\apisummary{
+  Exchanges a fixed amount of contiguous data blocks between all pairs
+  of \acp{PE} participating in the collective routine.
```

Comment:
Since we mention later that "All PEs in the provided team must participate in the collective.", can we just say that here, instead of keeping it in the abstract? 
i.e. "Exchanges a fixed amount of contiguous data blocks between all PEs on the specified team."